src/sentry/static/sentry/app/views/settings/components/forms/fieldFromConfig.jsx

```diff
@@ -85,7 +85,7 @@ export default class FieldFromConfig extends React.Component {
         // and rely on server validation
         delete props.required;
         // if (props.has_autocomplete) {
-        // return <Select2FieldAutocomplete {...props} />;
+        // return <SelectAutocompleteField {...props} />;
```

Comment:
Let's just remove these lines if it's not implemented

Reply:
I'd like to keep these because the next step will be converging all of these form components back together, but I'll add a TODO comment to clarify.